src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function (num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract: function (num) {
+      this.result -= num;
+
+      return this;
+    },
+    multiply: function (num) {
+      this.result *= num;
+
+      return this;
+    },
+    divide: function (num) {
+      this.result /= num;
+
+      return this;
+    },
+    reset: function () {
+      this.result = 0;
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - For object methods, the function keyword is not necessary. Use the method shorthand for a more modern and clean syntax.

Change from:
reset: function() {
  // your code
},

To:
reset() {
  // your code
},